package.json

```diff
@@ -24,8 +24,7 @@
     "styled-components": "^5.3.1",
     "suncalc": "^1.9.0",
     "ulid": "^2.3.0",
-    "web-vitals": "^1.0.1",
-    "winston": "^3.3.3"
+    "web-vitals": "^1.0.1"
```

Comment:
Can we remove this as well? It is an addition from CRA.

Reply:
Sure, good catch